sdk/documenttranslation/Azure.AI.DocumentTranslation/README.md

```diff
@@ -1,19 +1,548 @@
 # Azure Cognitive Services Document Translation client library for .NET
 
+Azure Cognitive Services Document Translation is a cloud service that translates documents to and from 90 languages and dialects while preserving document structure and data format. Use the client library for Document Translation to:
+
+* Translate numerous, large files from an Azure Blob Storage container to a target container in your language of choice.
+* Check the translation status and progress of each document in the translation operation.
+* Apply a custom translation model or glossaries to tailor translation to your specific case.
+
+[Source code][documenttranslation_client_src] | [Package (NuGet)][documenttranslation_nuget_package] | [API reference documentation][documenttranslation_refdocs] | [Product documentation][documenttranslation_docs] | [Samples][documenttranslation_samples]
+
 ## Getting started
 
 ### Install the package
+Install the Azure Document Translation client library for .NET with [NuGet][nuget]:
+
+```PowerShell
+dotnet add package Azure.AI.DocumentTranslation --prerelease
+```
 
 ### Prerequisites
+* An [Azure subscription][azure_sub].
+* An existing Translator resource.
+
+#### Create a Translator resource
+Document Translation supports [single-service access][single_service] only.
+To access the service, create a Translator resource.
+
+You can create either resource using: 
+
+**Option 1:** [Azure Portal][azure_portal_create_DT_resource].
+
+**Option 2:** [Azure CLI][cognitive_resource_cli]. 
+
+Below is an example of how you can create a Translator resource using the CLI:
+
+```PowerShell
+# Create a new resource group to hold the Translator resource -
+# if using an existing resource group, skip this step
+az group create --name <your-resource-name> --location <location>
+```
+
+```PowerShell
+# Create Translator
+az cognitiveservices account create \
+    --name <your-resource-name> \
+    --resource-group <your-resource-group-name> \
```

Comment:
@kristapratico made us the favor of testing this code, and notice that we need to explicitly add
`    --custom-domain document-translation-resource \`

Also, because currently the service only works on `S1` we should be specific about it here:
`--sku S1`